istioctl/pkg/writer/ztunnel/configdump/configdump.go

```diff
@@ -49,14 +55,76 @@ func (c *ConfigWriter) PrintBootstrapDump(outputFormat string) error {
 
 // PrintSecretDump prints just the secret config dump to the ConfigWriter stdout
 func (c *ConfigWriter) PrintSecretDump(outputFormat string) error {
-	// TODO
+	if c.ztunnelDump == nil {
+		return fmt.Errorf("config writer has not been primed")
+	}
+	secretDump := c.ztunnelDump.Certificates
+	out, err := json.MarshalIndent(secretDump, "", "    ")
+	if err != nil {
+		return fmt.Errorf("failed to marshal workloads: %v", err)
+	}
+	if outputFormat == "yaml" {
+		if out, err = yaml.JSONToYAML(out); err != nil {
+			return err
+		}
+	}
+	fmt.Fprintln(c.Stdout, string(out))
 	return nil
 }
 
 // PrintSecretSummary prints a summary of dynamic active secrets from the config dump
 func (c *ConfigWriter) PrintSecretSummary() error {
-	// TODO
-	return nil
+	if c.ztunnelDump == nil {
+		return fmt.Errorf("config writer has not been primed")
+	}
+	secretDump := c.ztunnelDump.Certificates
+	w := new(tabwriter.Writer).Init(c.Stdout, 0, 8, 5, ' ', 0)
+	fmt.Fprintln(w, "RESOURCE NAME\tTYPE\tSTATUS\tVALID CERT\tSERIAL NUMBER\tNOT AFTER\tNOT BEFORE")
```

Comment:
I think `RESOURCE NAME` can be just `NAME` to consistent with other config summaries.

Reply:
done